index.html

```diff
@@ -962,31 +837,46 @@ <h2>Examples</h2>
       <pre class="example highlight" title="Create a new blank exposed Thing">
         WoT.createExposedThing(thingDescription)
```

Comment:
Old API example. Should be `WoT.produce` without Promise and `then` etc

Also in addAction there is still old handler code.

Reply:
Thanks for the catch. Done.